c7n/resources/iam.py

```diff
@@ -2171,6 +2226,71 @@ def process(self, resources):
                         AccessKeyId=k['AccessKeyId'])
 
 
+@User.action_registry.register('remove-ssh-keys')
```

Comment:
nit s/remove/delete to match the api and permission names

Reply:
makes sense :+1: